justfile

```
@@ -156,7 +156,7 @@ ruff *args=".": devenv
 
 # run the various dev checks but does not change any files
 check: black django-upgrade ruff
-
+    $BIN/djhtml --tabwidth 2 --check templates
```

For consistency with the `black` and `ruff` recipes, we should have a `djhtml` recipe

```
djhtml *args="templates": devenv
    $BIN/djhtml --tabwidth 2 --check {{ args }}
```

which we then call from the `check` recipe

```suggestion
check: black django-upgrade ruff djhtml
```

---

excellent suggestion